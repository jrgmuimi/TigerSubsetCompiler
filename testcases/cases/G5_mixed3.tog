let
    var square : int
    var odd    : int
    var n      : int
in
    square  := 0;
    odd     := 1;
    for n:=1 to 101 do 
        square  := square+odd;
        odd     := odd + 2;
        if (square <= 100) then
            printint(square);
            printstring("\n")
        end
    end
end