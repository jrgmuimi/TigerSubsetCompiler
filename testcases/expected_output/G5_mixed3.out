MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

1
4
9
16
25
36
49
64
81
100

